src/pages/dev/send-tokens/interchain-tokens/create-token.mdx

```diff
@@ -2,7 +2,7 @@
 
 Interchain Tokens are ERC-20 tokens that are available on [multiple blockchains](https://docs.axelar.dev/resources/testnet). They are created using the [Interchain Token Service](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/InterchainTokenService.sol) and can be used to transfer value between blockchains.
 
-The Interchain Token Service is deployed to `0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C` while the Interchain Token Factory is deployed to `0x83a93500d23Fbc3e82B410aD07A6a9F7A0670D66`
+The Interchain Token Service is deployed to [`0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C`](https://blockscan.com/address/0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C) while the Interchain Token Factory is deployed to [`0x83a93500d23Fbc3e82B410aD07A6a9F7A0670D66`](https://blockscan.com/address/0x83a93500d23Fbc3e82B410aD07A6a9F7A0670D66)
```

Comment:
@ffe9f8 should we have a standard of linking to blockscan or ethereum for universal addresses? I'm leaning towards the latter but don't feel strongly.

Reply:
Discussed with @Olanetsoft, decided to have a standard of linking to etherscan.io because Blockscan can be viewed from the Etherscan interface.